doc/input-output-reference/src/overview/group-heating-and-cooling-coils.tex

```diff
@@ -1825,6 +1825,10 @@ \subsubsection{Inputs}\label{inputs-13-004}
        PLF = 1.0 + 0.0(PLR)
 \end{lstlisting}
 
+\paragraph{Field: Minimum Outdoor Dry-Bulb Temperature for Compressor Operation}\label{field-minimum-outdoor-dry-bulb-temperature-for-compressor-operation}
+
+This numeric field defines the minimum outdoor air dry-bulb temperature where the cooling coil compressor turns off. If this input field is left blank, the default value is -25 C.
+
```

Comment:
...input field is left blank, the value defaults to that specified in the IDD ??